community/kernel/src/main/java/org/neo4j/kernel/api/KernelTransaction.java

```diff
@@ -208,6 +211,25 @@ default void close() throws TransactionFailureException
 
     Revertable restrict( AccessMode mode );
 
+    /**
+     * @return list of executing queries that use this transaction
+     */
+    Stream<ExecutingQuery> executingQueries();
```

Comment:
Yes, I think so. It is totally possible in embedded to hold multiple undrained query results in the same tx, isn't it?